client/signup/steps/design-picker/index.jsx

```diff
@@ -174,6 +182,11 @@ export default function DesignPickerStep( props ) {
 						align="left"
 					/>
 				}
+				categoriesFooter={
+					useFeaturedPicksButtons && (
```

Comment:
Do we need to worry about adding a `useFeaturedPicksButtons` prop to the component? The DIFM designs don't include any featured picks, so we don't have to worry about disabling the Blank Canvas button in that flow.

```suggestion
					config.isEnabled( 'signup/design-picker-use-featured-picks-buttons' ) && !! featuredPicksDesigns.length && (
```

Reply:
Okay, I think we don't need this prop 👍 Besides, I've already checked `featuredPicksDesigns` inside `FeaturedPicksButtons`, so I think we don't need to check it here?